spec/integration/ssh_spec.rb

```diff
@@ -29,6 +29,13 @@
       expect(result['stdout'].strip).to eq(user)
     end
 
+    it 'expands options relative to current working directory' do
+      config_flags = %W[-t #{uri} --no-host-key-check -m #{modulepath} --private-key ./ssh/private --debug]
```

Comment:
This seems like it's duplicating the test in `spec/bolt/cli_spec.rb`. Is there something it's testing beyond what's already covered in the other test?

Reply:
:facepalm: Yup, removed